specification/vmware/resource-manager/Microsoft.AVS/preview/2019-08-09-preview/vmware.json

```diff
@@ -0,0 +1,1277 @@
+          }
+        },
+        "username": {
+          "description": "The ID of an Active Directory user with a minimum of read-only access to Base DN for users and group",
+          "type": "string"
+        },
+        "password": {
+          "description": "The password of the Active Directory user with a minimum of read-only access to Base DN for users and groups.",
+          "type": "string"
+        }
+      }
+    },
+    "PrivateCloud": {
+      "description": "A private cloud resource",
+      "allOf": [
+        {
+          "$ref": "#/definitions/TrackedResource"
+        }
+      ],
+      "properties": {
+        "sku": {
+          "description": "The private cloud SKU",
+          "$ref": "#/definitions/Sku"
+        },
+        "properties": {
+          "description": "The properties of a private cloud resource",
+          "$ref": "#/definitions/PrivateCloudProperties"
+        }
+      }
+    },
+    "PrivateCloudProperties": {
+      "description": "The properties of a private cloud resource",
+      "properties": {
+        "provisioningState": {
+          "description": "The provisioning state",
+          "type": "string",
+          "readOnly": true,
+          "enum": [
+            "Succeeded",
+            "Failed",
+            "Cancelled",
+            "Pending",
+            "Building",
+            "Updating"
+          ],
+          "x-ms-enum": {
+            "name": "PrivateCloudProvisioningState",
+            "modelAsString": true
+          }
+        },
+        "circuit": {
+          "description": "An ExpressRoute Circuit",
+          "$ref": "#/definitions/Circuit"
+        },
+        "cluster": {
+          "description": "The default cluster used for management",
+          "$ref": "#/definitions/DefaultClusterProperties"
+        },
+        "clusters": {
+          "description": "The clusters",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "type": "string"
+          }
+        },
+        "endpoints": {
+          "description": "The endpoints",
+          "readOnly": true,
+          "$ref": "#/definitions/Endpoints"
+        },
+        "internet": {
+          "description": "Connectivity to internet is enabled or disabled",
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "name": "InternetEnum",
+            "modelAsString": true
+          }
+        },
+        "identitySources": {
+          "description": "vCenter Single Sign On Identity Sources",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/IdentitySource"
+          }
+        },
+        "networkBlock": {
+          "description": "The block of addresses should be unique across VNet in your subscription as well as on-premise. Make sure the CIDR format is conformed to (A.B.C.D/X) where A,B,C,D are between 0 and 255, and X is between 0 and 22",
```

Comment:
nit: Just FYI, the x part is somewhat fluid as we had larger customer requested larger than usual block size which had forced us to make changes to the validation in SDK. **You may want to be intentionally a little vague here.** We had to went to a rang 0-32 for IPV4 and 0-64 for IPv6.

Reply:
I'll add this to the list for the next API version.